Intersect (Core)/Network/Packets/Server/NpcEntityPacket.cs

```diff
@@ -11,7 +11,7 @@ public NpcEntityPacket()
         }
 
 
-        [Key(24)]
+        [Key(25)]
```

Comment:
Same with this file..

Reply:
> Not looked at the full thing and functionality but just some quick stuff I saw scrolling through.
> 
> Also is this tested to work with asset packing?

We did not test it, I will do a test and respond again if it works or not with the packer.

Reply:
Since we needed an extra key for tags in both ends (Players and Npcs), we decided to add it as the 24th key on EntityPacket.cs. Is because of this that we had to change the keys of the Message Pack Objects in order to avoid a duplicated 24th key in the Entity packet structure.

Reply:
> 
> Also is this tested to work with asset packing?

Tested: this implementation has no issues with asset packing.